text/0000-base-image-label.md

```diff
@@ -0,0 +1,73 @@
+# Meta
+[meta]: #meta
+- Name: Add base image SHA to stack metadata
+- Start Date: 2020-10-23)
+- Author(s): dumez-k
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+Building off of this earlier [RFC](https://github.com/buildpacks/rfcs/blob/main/text/0050-stack-metadata.md) outlining stack metadata,
+we would like to add a label to expose the stack base image SHA in the metadata of all stack-cnb images.
+
+# Motivation
+[motivation]: #motivation
+
+- Allows stack users and authors to easily see which base-image that each stack cnb image is based off of.
+
+# What it is
+[what-it-is]: #what-it-is
+
+This provides a high level overview of the feature.
+
+- Define the target persona: stack user, stack author
+
+The label can be **optionally** added by stack maintainers to stack image:
+ 
+`io.buildpacks.stack.base_image:` SHA of stack base image
+
+
+# How it Works
+[how-it-works]: #how-it-works
+
+This is the technical portion of the RFC, where you explain the design in sufficient detail.
+
+The `base_image` label will be added as a `Label` in the stack's Dockerfile.
+Example:
+
+`LABEL io.buildpacks.stack.base_image=paketobuildpacks/build@sha256:e3c1c2659c2468ea8ac8058c501293882b0a55622840eee53525f90b59d31a33`
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+One downside is it adds slightly more output to `inspect-image` commands which some users may find extraneous.
```

Comment:
How do you expect this to be reported by `pack inspect-image`? Currently this output include a base image, which is the build-image (the build-image is the base of the app image)

e.g.
```
Base Image:
  Reference: 523e5e0ad0892f136eebebcaaa6fd60cd744d9b449ade675c3d2a4cad9c4de3e
  Top Layer: sha256:83c184a138eb51d80858fed5f484dbf0cf62aff327ba4c9364f5c81eb5bd0303
```

Reply:
Really good point that the "stack base image" isn't an established domain concept. Completely agree and seeing as we can as you pointed out add arbitrary labels anyway, going to go ahead and close this RFC. 

Thanks @ekcasey !